Makefile

```diff
@@ -71,11 +80,14 @@ ifeq ($(USE_MKLDNN), 1)
 	export USE_MKLML = 1
 endif
 
+
 include $(TPARTYDIR)/mshadow/make/mshadow.mk
 include $(DMLC_CORE)/make/dmlc.mk
 
+include 3rdparty/ngraph-mxnet-bridge/ngraph.mk
+
 # all tge possible warning tread
-WARNFLAGS= -Wall -Wsign-compare
+WARNFLAGS= -Wall -Wsign-compare -Wno-comment
```

Comment:
can we try to limit our changes to ngraph submodule , unless we need to

Reply:
when we include ngraph.hpp in ndarray.h, it prints a lot of warnings from doxygen code comments for anything that includes ndarray.h